bin/version2/waagent

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+#
```

Comment:
i'd suggest naming the new directory 'py3' or something like that instead of 'v2'

Reply:
addressed